corehq/apps/translations/integrations/transifex/parser.py

```diff
@@ -87,8 +87,10 @@ def _add_module_sheet(self, ws, po_entries):
         ws.append(["case_property", "list_or_detail", self.key_lang_str, self.source_lang_str])
         for po_entry in self._get_rows_for_module_sheet(po_entries):
             context = po_entry.msgctxt
-            _case_property, _list_or_detail = re.match(context_regex, context).groups()
-            ws.append([_case_property, _list_or_detail, po_entry.msgid, po_entry.msgstr])
+            regex_match = re.match(context_regex, context)
+            if regex_match is not None:
```

Comment:
Does it fall back to the default language when it cannot find translation?

Reply:
If I'm understanding correctly, here you are downloading the translations that are in your transifex project already. With this change, in the case that no translation is found for a property that row in the spreadsheet isn't written. You shouldn't get an error if your translations for a particular language are present in your project, but QA was hitting a NoneType attr error at `regex_match.groups()`, possibly because of incomplete translations in the project.
I think the checkbox `Confirm that resources are completely translated before performing request` is in place to make sure that you don't try to download resources that don't exist on you project. @mkangia does that sounds correct?

Reply:
Hey @Robert-Costello 

I believe we don't expect here to not have a match.
Since the translations that are being parsed, where originally pushed by HQ only, all translations should match the relevant regex since HQ adds the context on all source strings when they are pushed to transifex.

Even if source string has not been translated, this should still be okay since the context would still be present.

So, looks like QA did something else. Did they add a string manually to transifex? Good to check on what the context for the failing translation was and see why that was not matching the expected regex.

As for the checkbox, you are right that it is used to first verify that translations are completed but that shoud not related to the error here.

Reply:
Thanks @mkangia
I see what you mean about the context being added by HQ should mean that we get a match. I reached out to QA for some more info.

Reply:
Confirmed that some strings were added directly to transifex. I relayed to QA that adding those strings manually is the cause of the `NoneType` error since the context that's normally added by hq was missing. 
Ideally items will not be added directly to transifex in the future. I'd say this is still a good check to have so that in case that does happen again it will not completely block work as it did for BHA testing in this instance.